### DISCUSSION TITLE
Bonus from the collection of bears by quantity

### DISCUSSION THREAD

**Discussion by JCtapuk** [sunflower-land/sunflower-land#2679] [Ideas]
I noticed that many players like to collect a collection of bears. I propose to strengthen the dependence on the number of bears.

Level 1: Need to collect 5 or more bears and get a +0.1 crop bonus.
Level 2: You need to collect 10 or more bears. Grants a level 1 bonus and a 5% growth rate bonus.
Level 3: You need to collect 15 bears or more. Gives 1, 2 bonus levels and 5% more sfl sales.

the more bears, the better bonuses it gives. I don’t know yet how many bears are available in the game, but as the project develops, the bears become more decorative.

**Comment by brian-man** [comment]
I disagree.  Decorations should be decorations.  Not "decoration but someday maybe I'll get some boost from them".  This leads to players "overbuying" decorations and then throwing tantrums all over Discord because their speculative plays didn't pan out.

**Comment by adamhannigan** [comment]
Thanks for raising this suggestion @JCtapuk 

For the time being we are keeping the bears as decorative content. As @brian-man raised we need to be careful setting player expectations around certain items.

The tier system you mention is interesting, collect X amount of items for Level 1, 2 & 3 buffs. In this case I would suggest a feature like this gets implemented with new items that are have a clear purpose of giving a buff
